roles/nginx/defaults/main.yml

```diff
@@ -16,3 +16,8 @@ nginx_cache_path: /var/cache/nginx
 nginx_cache_key_storage_size: 10m
 nginx_cache_size: 250m
 nginx_cache_inactive: 1h
+
+
+# This should be turned on if you are using `gzip_static_assets_on_deploy`.
+# This value should be qouted.
```

Comment:
Misspelling, it should be "quoted," but I feel like we should just remove the comments and nginx_gzip_static option. Simply set gzip_static to on if gzip_static_assets_on_deploy is set to true. Thoughts?